docs/how-to-run.adoc

```diff
@@ -122,9 +120,17 @@ may be executed:
 
 [source,text]
 ----
-$ pfexec cargo run --release --bin omicron-package -- install
+$ cargo build --release --bin omicron-package
+$ pfexec ./target/release/omicron-package install
 ----
 
+[NOTE]
+====
+**Do not use `pfexec cargo run` directly**; it will cause files in `~/.cargo` and `target/` to be owned by root, which will cause problems down the road.
+
+If you've done this already, and you wish to recover, run from the root of this repository `pfexec chown -R $USER:$(id -ng $USER) target ${CARGO_HOME:-~/.cargo}`.
```

Comment:
Good catch, and sorry about this. I have definitely run into this issue before too...